src/test/java/core/basesyntax/report/ReportRecorderImplTest.java

```diff
@@ -2,39 +2,29 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import core.basesyntax.model.FruitTransaction;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import org.junit.jupiter.api.Test;
 
 class ReportRecorderImplTest {
-    private static final ReportRecorder REPORT_RECORDER = new ReportRecorderImpl();
-    private static final Map<String, Integer> VALID_MAP = new HashMap<>();
+    private static ReportRecorder reportRecorder = new ReportRecorderImpl();
```

Comment:
Initialize it in @BeforeAll, or do not remove `final` keyword if you are initializing it immediately